src/compile/common.ts

```diff
@@ -18,17 +18,18 @@ import {RepeaterValue, RepeatModel} from './repeat';
 import {UnitModel} from './unit';
 
 
-export function buildModel(spec: Spec, parent: Model, parentGivenName: string, repeater: RepeaterValue, config: Config): Model {
+export function buildModel(spec: Spec, parent: Model, parentGivenName: string,
```

Comment:
I wonder whether we should just put the size in the config. No need to add another parameter everywhere.